test/src/lib.rs

```diff
@@ -178,9 +178,8 @@ mod tests {
         mark_architecture_skipped(&mut tests, "mips64el"); // TODO: Check reason for failure!
         mark_architecture_skipped(&mut tests, "mips"); // TODO: Check reason for failure!
         mark_architecture_skipped(&mut tests, "mipsel"); // TODO: Check reason for failure!
-        mark_architecture_skipped(&mut tests, "ppc64"); // TODO: Check reason for failure!
-        mark_architecture_skipped(&mut tests, "ppc64le"); // TODO: Check reason for failure!
-        mark_architecture_skipped(&mut tests, "ppc"); // TODO: Check reason for failure!
+        mark_architecture_skipped(&mut tests, "ppc64"); // Ghidra generates mangled function names here for some reason.
+        mark_architecture_skipped(&mut tests, "ppc64le"); // Ghidra generates mangled function names here for some reason.
```

Comment:
So we don't skip 'ppc' anymore?

Reply:
Yes. :-) The problem with 'ppc' was a trivial expression of the form `a OR a`, that the analysis could not handle. By substituting it away, the check now works for 'ppc'.